modules/meta.analysis/tests/Rcheck_reference.log

```diff
@@ -36,113 +81,41 @@ address and non-empty name.
-approx.posterior: no visible global function definition for ‘pdf’
-approx.posterior: no visible global function definition for ‘var’
-approx.posterior: no visible global function definition for ‘plot’
-approx.posterior: no visible global function definition for ‘density’
-approx.posterior: no visible global function definition for ‘rug’
-approx.posterior: no visible global function definition for ‘lines’
-approx.posterior: no visible global function definition for ‘dbeta’
-approx.posterior: no visible global function definition for ‘legend’
-approx.posterior: no visible binding for global variable ‘AIC’
-approx.posterior: no visible global function definition for ‘sd’
-approx.posterior: no visible global function definition for ‘dev.off’
 jagify: no visible binding for global variable ‘stat’
 jagify: no visible binding for global variable ‘n’
 jagify: no visible binding for global variable ‘site_id’
 jagify: no visible binding for global variable ‘greenhouse’
 jagify: no visible binding for global variable ‘citation_id’
 jagify: no visible binding for global variable ‘name’
-pecan.ma: no visible global function definition for ‘stem’
-pecan.ma: no visible global function definition for ‘window’
-pecan.ma.summary: no visible global function definition for
-  ‘is.mcmc.list’
-pecan.ma.summary: no visible global function definition for ‘theme_set’
-pecan.ma.summary: no visible global function definition for ‘theme_bw’
-pecan.ma.summary: no visible global function definition for ‘ggmcmc’
-pecan.ma.summary: no visible global function definition for ‘ggs’
-pecan.ma.summary: no visible global function definition for ‘pdf’
-pecan.ma.summary: no visible global function definition for ‘plot’
-pecan.ma.summary: no visible global function definition for ‘box’
-pecan.ma.summary: no visible global function definition for ‘dev.off’
-run.meta.analysis: no visible global function definition for ‘db.open’
-run.meta.analysis: no visible global function definition for ‘db.close’
-run.meta.analysis.pft: no visible binding for global variable
-  ‘settings’
-run.meta.analysis.pft: no visible binding for global variable
-  ‘trait.data’
-run.meta.analysis.pft: no visible global function definition for
-  ‘median’
-run.meta.analysis.pft: no visible binding for global variable
-  ‘prior.distns’
-run.meta.analysis.pft: no visible global function definition for
-  ‘dbfile.insert’
 Undefined global functions or variables:
-  AIC box citation_id db.close db.open dbeta dbfile.insert density
-  dev.off ggmcmc ggs greenhouse is.mcmc.list legend lines median n pdf
-  plot prior.distns rug sd settings site_id stat stem theme_bw
-  theme_set trait.data var window
-Consider adding
-  importFrom("graphics", "box", "legend", "lines", "plot", "rug", "stem")
-  importFrom("grDevices", "dev.off", "pdf")
-  importFrom("stats", "AIC", "dbeta", "density", "median", "sd", "var",
-             "window")
-to your NAMESPACE file.
+  citation_id greenhouse n name site_id stat
```

Comment:
note that `name` wasn't in the previous list. Does this mean that a recent PR introduced a new undefined global? @infotroph how did that sneak past you code?

Reply:
@mdietze In line 73 `name` was there as a global variable, but it is fixed in PR #2973, will change the `Rcheck_reference.log` file when the PR is merged